configs/detection/ava/slowfast_context_kinetics_pretrained_r50_4x16x1_20e_ava_rgb.py

```diff
@@ -159,7 +159,12 @@
 total_epochs = 20
 checkpoint_config = dict(interval=1)
 workflow = [('train', 1)]
-evaluation = dict(interval=1)
+# Three avaiable key indicators for ava dataset
+# 'Recall@0.5@100' 'AR@100' 'PascalBoxes_Precision/mAP@0.5IOU'
```

Comment:
Where does the name `PascalBoxes_Precision` come from?

Reply:
I think this comes from official ava evalution codes.

https://github.com/open-mmlab/mmaction2/blob/023777cfd26bb175f85d78c455f6869673e0aa09/mmaction/core/evaluation/ava_evaluation/object_detection_evaluation.py#L307-L308

And `self._metric_prefix` is defined 

https://github.com/open-mmlab/mmaction2/blob/023777cfd26bb175f85d78c455f6869673e0aa09/mmaction/core/evaluation/ava_evaluation/object_detection_evaluation.py#L151

https://github.com/open-mmlab/mmaction2/blob/023777cfd26bb175f85d78c455f6869673e0aa09/mmaction/core/evaluation/ava_evaluation/object_detection_evaluation.py#L356